GliaWidgets/Theme/Survey/Theme+Survey.swift

```diff
@@ -0,0 +1,112 @@
+import Foundation
+import UIKit
+
+public extension Theme {
+    /// Text style.
+    struct Text {
```

Comment:
I suppose you will need to provide public initializers for this and similar structs, since automatically synthesized will have internal access modifier by default, thus unavailable for integrator.

Reply:
I thought that this is not needed in sense of a big initializer and integrators will change default values instead of implementing the full object separately, but anyway it makes sense to provide an initializer.
fixed - 2720cc15d186e3ab1ba91d1d759f26515e32142b